modules/ipc/methods/eth_sendTransaction.js

```diff
@@ -36,14 +36,16 @@ module.exports = class extends BaseProcessor {
             try {
                 _.each(payload.params[0], (val, key) => {
                     // if doesn't have hex then leave
-                    if (_.isString(val)) {
+                    if (_.isString(val) || _.isNumber(val)) {
```

Comment:
Technically all payload should be he. So please use a Regex `/[a-f0-9]/i.test(val)` and test if it works.